packages/validation-manager/tsconfig.packages.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
why we need 2 tsconfig files?

Reply:
I am not sure but this is needed for yarn workspaces. Removing these used to break the project in the past, did not try since.